cmd/ebrelayer/Dockerfile

```diff
@@ -1,21 +1,23 @@
 #
 # Build
 #
-FROM golang:1.15.2 AS build
+FROM golang:1.15 AS build
 
 ENV GOBIN=/go/bin
 ENV GOPATH=/go
-ENV CGO_ENABLED=1
+ENV CGO_ENABLED=0
```

Comment:
why is `ENV CGO_ENABLED=0` needed?

Reply:
personally I think this should be removed.

Reply:
@utx0 Yeah, we could remove it actually. I changed this to `0` as the compiled binary previously had issues (wouldn't run).